bcgov/PIMS
# PIMS-2220 Project Properties Map

## 🎯 Summary

  
[PIMS-2220](https://citz-imb.atlassian.net/browse/PIMS-2220)


## Changes
- Reworked the cluster popup so it will centre correctly no matter what size of map it's on
- Added additional configuration options to the parcel map
- Added a parcel map to the project details page
- Added parcel boundary details to maps by default
- PID/Address values in the disposal properties tables are now links to the properties
- Fixed an issue with the PropertyRow not showing the PIN correctly if the property had a PID with value 0.

## Testing
- Navigate to a project with associated properties
- See that the map is present and behaves as expected. Should have a cluster popup and some zoom/drag controls
- Check the main map, property add map, and property details map to see that they also still work as expected.


## 🔰 Checklist


> - I have performed a self-review of my code.
> - I have commented my code, particularly in hard-to-understand areas.
> - I have made corresponding changes to the documentation where required.
> - I have tested my changes to the best of my ability.
> - My changes generate no new warnings.
